templates/skeleton/mobile/source/class/custom/page/Login.tmpl.js

```diff
@@ -0,0 +1,67 @@
+/* ************************************************************************
+
+   Copyright:
```

Comment:
Would you add the header variables (author, copyright etc.) here too, so all the files are consistent?